src/styles/typeface/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Typeface
+description: If your service is on the service.gov.uk subdomain you must use the GDS Transport font
+section: Styles
+theme: Typography
+layout: layout-pane.njk
+order: 5
+# headings:
+#   - text: Section break
+#     aliases: horizontal rule, hr
+
+#   - text: Links
+#     ignoreInPageNav: true
+#     ignoreInSearch: true
+
+#   - text: Lists
+#     ignoreInPageNav: true
+#     ignoreInSearch: true
+---
+
+{% from "_example.njk" import example %}
+
+
+If your service is on the service.gov.uk subdomain you must use the GDS Transport font.
```

Comment:
This only covers one of the [allowed domains](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk). I don't think it's useful to replicate content but maybe this page should just be a wrapper for that service manual one?

Reply:
Decision: leave it for now because it's a complex problem